skrub/_tabular_learner.py

```diff
@@ -129,34 +136,34 @@ def tabular_learner(estimator, n_jobs=None):
                     ('simpleimputer', SimpleImputer(add_indicator=True)),
                     ('logisticregression', LogisticRegression())])
 
-    By applying only the first pipeline step we can see the transformed data
-    that is sent to the supervised estimator (see the :obj:`TableVectorizer`
-    documentation for details):
+    By applying only the first pipeline step we can see the transformed data that is
+    sent to the supervised estimator (see the :obj:`TableVectorizer` documentation for
+    details):
 
-    >>> model.named_steps['tablevectorizer'].transform(X)            # doctest: +SKIP
+    >>> model.named_steps['tablevectorizer'].transform(X)
        last_visit_year  last_visit_month  ...  insulin_prescriptions  fasting_glucose
     0           2020.0               1.0  ...                    NaN             35.0
     1           2021.0               4.0  ...                   13.0            140.0
     2           2024.0              12.0  ...                    0.0             44.0
     3           2023.0               8.0  ...                   17.0            137.0
 
-    The parameters of the :obj:`TableVectorizer` depend on the provided
-    ``estimator``.
+    The parameters of the :obj:`TableVectorizer` depend on the provided ``estimator``.
 
     >>> tabular_learner(LogisticRegression())
     Pipeline(steps=[('tablevectorizer', TableVectorizer()),
                     ('simpleimputer', SimpleImputer(add_indicator=True)),
                     ('logisticregression', LogisticRegression())])
 
-    We see that for the :obj:`~sklearn.linear_model.LogisticRegression` we get
-    the default configuration of the ``TableVectorizer`` which is intended to
-    work well for a wide variety of downstream estimators. Moreover, as the
-    ``LogisticRegression`` cannot handle missing values, an imputation step is
-    added.
+    We see that for the :obj:`~sklearn.linear_model.LogisticRegression` we get the
```

Comment:
I wondering if the discussion here and below should not go directly to the user guide.

My own habit is to use this section to quickly copy pasting code to get started. If I want to get inside I would naturally go read the information in the user guide.

I was wondering if we want a similar pattern or this is just me?